CommonMC/src/SelectRecoMC_module.cc

```diff
@@ -310,15 +310,15 @@ namespace mu2e {
           tshmc._cpos, tdir );
 
     auto mcsep = wirepca.point2() - wirepca.point1();
-    tshmc._wireLen = mcsep.Dot(wdir);
+    tshmc._wireLen = (wirepca.point2()-XYZVectorF(straw.wirePosition())).Dot(wdir);
```

Comment:
I don't understand how wirepca.point1() could be off the wire position.  Is one of those really WRT  the straw axis?  If so, can the names be set so that it's more clear?

Reply:
wirepca.point1() is at the same longitudinal position so mcsep.Dot(wdir) ~ 0, need to compare to the wire middle instead